src/locale/locales/fr/messages.po

```diff
@@ -5203,39 +4839,31 @@ msgstr "Désépingler"
 
 #: src/view/screens/ProfileFeed.tsx:303
 msgid "Unpin from home"
-msgstr ""
+msgstr "Désépingler de l’accueil"
 
 #: src/view/screens/ProfileList.tsx:446
 msgid "Unpin moderation list"
 msgstr "Supprimer la liste de modération"
 
-#: src/view/screens/ProfileFeed.tsx:346
-#~ msgid "Unsave"
-#~ msgstr "Supprimer"
-
 #: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:225
 msgid "Unsubscribe"
-msgstr ""
+msgstr "Désabonner"
```

Comment:
Unless you force someone else to `unsubscribe`, you have to add the `se` as it's a pronominal verb.

```suggestion
msgstr "Se désabonner"
```